checker/cost.go

```diff
@@ -533,14 +533,34 @@ func (c *coster) functionCost(function, overloadID string, target *AstNode, args
 
 	if est := c.estimator.EstimateCallCost(function, overloadID, target, args); est != nil {
 		callEst := *est
-		return CallEstimate{CostEstimate: callEst.Add(argCostSum())}
+		return CallEstimate{CostEstimate: callEst.Add(argCostSum()), ResultSize: est.ResultSize}
```

Comment:
Would helper methods to create size and cost based estimates help avoid missing sizes in the future?

Reply:
Yeah, I think it would. How about add it as follow up so we can backport this as a minimal fix?